industry/products.md

```diff
@@ -12,16 +12,19 @@ They vary by:
 - accuracy
 - price
 - customisation options
-- data confidentiality
-
+- [data confidentiality](data-confidentiality.md)
 and other features, as well as underlying architecture.
 
+Some providers offer ready-to-use solutions that can be integrated immediately.  These usually support many language pairs and their infrastructure is scalable.  An alternative is to train a model from scratch — or to use a pre-trained model — and deploy and maintain it.
```

Comment:
I would really prefer "systems" to "solutions" here, just more eng and less enterprisy.  (We have to listen to that stuff enough in our day jobs. ;-))

Reply:
We didn't straight up say what exactly it is, that it's an API or something like that.